test/e2e/configurations/ci_e2e_config_hetzner.yaml

```diff
@@ -0,0 +1,34 @@
+# configuration/parameters for CI/CD e2e test runs
+pvcstress:
+  replicas: 1
+  cdCycles: 100
+  crudCycles: 10
+ioSoakTest:
+  replicas: 1
+  duration: 10m
+  loadFactor: 2
+  protocols:
+  - nvmf
+  - iscsi
```

Comment:
This was an oversight on my path in the original configuration, for the moment we should only check nvmf so remove iscsi ?

Reply:
will do